calyx/frontend/src/workspace.rs

```diff
@@ -79,16 +80,33 @@ impl Workspace {
             return Ok(relative_import);
         }
 
-        let library_import = lib_path.join(import.clone());
-        if library_import.exists() {
-            return Ok(library_import);
+        let library_imports: Vec<_> = lib_paths
+            .iter()
+            .filter_map(|lib_path| {
+                let library_import = lib_path.join(import.clone());
+                library_import.exists().then_some(library_import)
+            })
+            .collect();
+        if library_imports.len() == 1 {
+            return Ok(library_imports.into_iter().next().unwrap());
+        }
+        if library_imports.len() > 1 {
+            return Err(Error::misc(format!(
+                "Import path `{}` found in multiple library paths ({})",
+                import.as_ref().to_string_lossy(),
+                library_imports
+                    .iter()
+                    .map(|p| p.to_string_lossy())
+                    .format(", ")
+            ))
+            .with_pos(&import));
```

Comment:
Not super important, but I think these blocks could probably be arms of a match
on the len of library imports rather than a bunch of if with returns. Might be a
bit easier to parse?